packages/context-menu/src/vaadin-contextmenu-event.js

```diff
@@ -3,7 +3,8 @@
  * Copyright (c) 2016 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { isIOS } from '@vaadin/component-base/src/browser-utils.js';
+import { isKeyboardActive } from '@vaadin/a11y-base';
```

Comment:
Consider using the full path for consistency:

```suggestion
import { isKeyboardActive } from '@vaadin/a11y-base/src/focus-utils.js';
```